src/D2L.Bmx/LoginHandler.cs

```diff
@@ -0,0 +1,13 @@
+namespace D2L.Bmx;
+
+internal class LoginHandler(
+	OktaAuthenticator oktaAuth
+) {
+	public async Task HandleAsync(
+		string? org,
+		string? user
+	) {
+		await oktaAuth.AuthenticateAsync( org, user, nonInteractive: false );
```

Comment:
We should check if the config file exists first. If not, Okta session won't be saved, and there's no point in running login. We should exit with error and warn the user about this.

Reply:
Good idea